src/test/java/com/spanish_inquisition/battleship/client/network/ResponsesBusTest.java

```diff
@@ -0,0 +1,38 @@
+package com.spanish_inquisition.battleship.client.network;
+
+import com.spanish_inquisition.battleship.common.Header;
+import com.spanish_inquisition.battleship.common.NetworkMessage;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * @author Michal_Partacz
+ */
+public class ResponsesBusTest {
+
+    @Test
+    public void testHasServerResponses() throws Exception {
```

Comment:
name should be like: "shouldAddResponseToEmptyBus".

Not use "test" in test methods, because @Test annotations tell that it is a test.